examples/step-75/step-75.cc

```diff
@@ -0,0 +1,1575 @@
+    Assert(solver_type == SolverType::Matrix, ExcNotImplemented());
+
+    TimerOutput::Scope t(computing_timer, "init");
+
+    Assert(min_level <= max_level,
+           ExcMessage(
+             "Triangulation level limits have been incorrectly set up."));
+    // TODO: move min/max degrees to parameterhandler
+    const unsigned int min_degree = 2, max_degree = dim <= 2 ? 7 : 5;
+    Assert(min_degree <= max_degree,
+           ExcMessage("FECollection degrees have been incorrectly set up."));
+
+    mapping_collection.push_back(MappingQ1<dim>());
+
+    for (unsigned int degree = min_degree; degree <= max_degree; ++degree)
+      {
+        fe_collection.push_back(FE_Q<dim>(degree));
+        quadrature_collection.push_back(QGauss<dim>(degree + 1));
+      }
+
+    fe_values_collection =
+      std::make_unique<hp::FEValues<dim>>(fe_collection,
+                                          quadrature_collection,
+                                          update_gradients |
+                                            update_quadrature_points |
+                                            update_JxW_values);
+    fe_values_collection->precalculate_fe_values();
+
+    switch (adaptation_type)
+      {
+        case AdaptationType::h:
+          adaptation_strategy =
+            std::make_unique<AdaptationStrategy<dim>>(ParameterHandler(),
+                                                      dof_handler,
+                                                      triangulation);
+          break;
+        case AdaptationType::hpLegendre:
+          adaptation_strategy = std::make_unique<hpLegendreStrategy<dim>>(
+            ParameterHandler(), dof_handler, triangulation, fe_collection);
+          break;
+        case AdaptationType::hpFourier:
+          adaptation_strategy = std::make_unique<hpFourierStrategy<dim>>(
+            ParameterHandler(), dof_handler, triangulation, fe_collection);
+          break;
+        case AdaptationType::hpHistory:
+          adaptation_strategy =
+            std::make_unique<hpHistoryStrategy<dim>>(ParameterHandler(),
+                                                     dof_handler,
+                                                     triangulation);
+          break;
+        default:
+          AssertThrow(false, ExcNotImplemented());
+          break;
+      }
+  }
+
+
+
+  template <int dim>
+  void LaplaceProblem<dim>::create_coarse_grid()
+  {
+    TimerOutput::Scope t(computing_timer, "coarse grid");
+
+    std::vector<unsigned int> repetitions(dim, 2);
+    Point<dim>                bottom_left, top_right;
+    for (unsigned int d = 0; d < dim; ++d)
+      {
+        bottom_left[d] = -1.;
+        top_right[d]   = 1.;
+      }
+
+    std::vector<int> cells_to_remove(dim, 1);
+    cells_to_remove[0] = -1;
+
+    // TODO
+    // expand domain by 1 cell in z direction for 3d case
+
+    GridGenerator::subdivided_hyper_L(
+      triangulation, repetitions, bottom_left, top_right, cells_to_remove);
```

Comment:
Is this not what you would get from `hyper_L` and then one global refinement?

Reply:
`hyper_L` removes the I. quadrant, while this implementation removes the IV. quadrant.

One could argue that it doesn't matter which quadrant will be removed. However on this domain, the `Solution` function is easier to comprehend. (Otherwise we would need to introduce an azimuth shift of `pi/2`.)

I would like to keep it like this to demonstrate how to use `GridGenerator::subdivided_hyper_L`.